pkgs/by-name/ne/nexusmods-app/package.nix

```
@@ -51,9 +51,12 @@ buildDotnetModule rec {
     ln --force --symbolic "${lib.getExe _7zzWithOptionalUnfreeRarSupport}" src/ArchiveManagement/NexusMods.FileExtractor/runtimes/linux-x64/native/7zz
   '';
 
+  NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR = 1; # Use nixpkgs 7zz executable
+
   makeWrapperArgs = [
     "--prefix PATH : ${lib.makeBinPath [desktop-file-utils]}"
     "--set APPIMAGE $out/bin/${meta.mainProgram}" # Make associating with nxm links work on Linux
+    "--set INSTALLATION_METHOD_PACKAGE_MANAGER 1" # Disable integrated updater
```

Technically, there is no integrated updater. Only an update _checker/notifier_.

This env doesn't disable anything, rather it changes the user-facing instructions to something along the lines of _"update available. Use your system package manager to install it."_ (I haven't looked at the actual wording, this is based off of the docs & and discord conversations).

```suggestion
    "--set INSTALLATION_METHOD_PACKAGE_MANAGER 1" # Tell the app it's a distro package
```